<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="styleSheet">
      <string notr="true">QSplitter::handle {
    width: 5px; /* 设置分割线的宽度 */
}</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>存储相关</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QPushButton" name="pbnIsNullEmpty">
            <property name="text">
             <string>isNull &amp;&amp; isEmpty</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pbnResize">
            <property name="text">
             <string>resize</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pbnSizeLength">
            <property name="text">
             <string>size &amp;&amp; length</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pbnFill">
            <property name="text">
             <string>fill</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>字符串截取</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="pbnFrontBack">
            <property name="text">
             <string>front &amp;&amp; back</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pbnLeftRight">
            <property name="text">
             <string>left &amp;&amp; right</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pbnFirstLast">
            <property name="text">
             <string>first &amp;&amp; last</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pbnSection">
            <property name="text">
             <string>section *</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>搜索和判断</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QPushButton" name="pbnIndexLastOf">
            <property name="text">
             <string>indexOf &amp;&amp; lastIndexOf</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pbnContains">
            <property name="text">
             <string>contains</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pbnEndStartWith">
            <property name="text">
             <string>endsWith &amp;&amp; startsWith</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pbnCount">
            <property name="text">
             <string>count</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>转换和修改</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QPushButton" name="pbnTrimSimplfy">
            <property name="text">
             <string>trimmed &amp;&amp; simplified</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pbnInsert">
            <property name="text">
             <string>insert</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pbnRemove">
            <property name="text">
             <string>remove</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pbnReplace">
            <property name="text">
             <string>replace</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>GroupBox</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pbnClear">
            <property name="text">
             <string>清空文本框</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="pteContent"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>输入</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>str2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>str1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cbxStr2">
            <item>
             <property name="text">
              <string>\</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Qt</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>.cpp</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>:</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>，</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QSpinBox" name="spinBox"/>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QComboBox" name="cbxStr1">
            <item>
             <property name="text">
              <string>G:\Qt6Book\QtSamples</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>G:\Qt6Book\QtSamples\qw.cpp</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>   Are     you   OK    ?     </string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>学生姓名，男，2003-6-15，汉族，山东</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>四海八方</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Dimple</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string> </string>
             </property>
            </item>
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>参数</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
